src/data-dictionary/events/K8sContainerSample/aa_template.md

```diff
@@ -0,0 +1,9 @@
+---
+name: 
+type: attribute
+units:
+events:
+  -  K8sContainerSample
+---
+
```

Comment:
missing a description. this prob just needs a deletion

Reply:
Yup, this was deleted since it was just a template we forgot to remove.